.github/workflows/build_test.yml

```diff
@@ -0,0 +1,41 @@
+name: ROS2 CI
+
+on:
+  pull_request:
+    branches:
+      - 'ros2/*'
+      - 'ros2'
```

Comment:
Would adding push here make it so the github actions are visible in the PR prior to a merge?  (I'm also curious if there's a repo admin setting or even a `ros-drivers` wide setting that prevents any github actions running)

```
  push:
    branches:
      - 'ros2/*'
      - 'ros2'
      ```

Reply:
`push` events happen after a merge. the reason why they arent being triggered for this PR is a security feature of github actually. since they have never been run before on this repo and im make a PR from a forked repo it wont allow them to be run.  its to prevent random people making PR's to your repos and then causing some action to take place i think

Reply:
actions are set to `enabled` currently on this repo - I can see that now that i am an admin. once we merge this - then more PR's from people to any branch that starts with `ros2` should trigger the CI.